indico/modules/events/views.py

```diff
@@ -111,6 +111,7 @@ def _getBody(self, params):
                              category=self.event.category.title,
                              timezone=self.event.display_tzinfo,
                              theme_settings=self.theme.get('settings', {}),
+                             theme_user_settings=self.theme.get('user_settings', {}),
```

Comment:
That's not what I meant - the settings definitions are quire useless in the template. Instead, get the actual settings dict here (from the layout_settings) - but only if the theme is the event's theme (ie not if it's just selected via `?view=...`)